src/main/java/hudson/plugins/sonar/action/SonarProjectActionFactory.java

```diff
@@ -57,7 +57,7 @@ public SonarProjectActionFactory() {
 
   @Override
   public Collection<? extends Action> createFor(AbstractProject project) {
-    if (!projectHasSonarAnalysis(project)) {
+    if (project == null || !projectHasSonarAnalysis(project)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2583)